### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by neiltuscano** [lllyasviel/Fooocus#3277] [Q&A] [answered]
aise err
OSError: [WinError 126] The specified module could not be found. Error loading "C:\Users\NEILT\Desktop\Fooocus_win64_2-1-831\python_embeded\lib\site-packages\torch\lib\c10.dll" or one of its dependencies.

**Comment by neiltuscano** [comment]
installed VS redis worked for me

**Reply by mashb1t** [reply, marked as answer]
This is unrelated, Fooocus does not use and does not have references to redis at all.
